fetch.bs

```diff
@@ -7178,6 +7174,22 @@ returns failure or a <a for=/>MIME type</a>.
 
 <hr>
 
+<div algorithm>
+<p>To <dfn id=concept-body-get-mime-type for=Body>get the MIME type</dfn>, given a {{Request}}
+or {{Response}} object, <var>requestOrResponse</var>, run these steps:
+
+<ol>
+ <li><p>Let <var>headers</var> be null
+ <li><p>If <var>requestOrResponse</var> is a {{Request}} object, then set <var>headers</var> to
+ <var>requestOrResponse</var>'s <a for=Request>request</a>'s {{Request/headers}}.
+ <li><p>Otherwise, set <var>headers</var> to <var>requestOrResponse</var>'s <a for=Response>
+ response</a>'s {{Response/headers}}.
+ <li><p>Let <var>mimeTypeInHeaders</var> be the result of <a for="header list">extracting a
+ MIME type</a> from <var>headers</var>.
```

Comment:
No newlines in phrasing-level elements to make inline search not require a regex.

Reply:
Understood! Thanks for catching it. Is following style acceptable?

```html
<li><p>Let <var>mimeTypeInHeaders</var> be the result of <a for="header list">
 extracting a MIME type</a> from <var>headers</var>.
```

Reply:
No, the whole element needs to go on on the new line (otherwise you introduce whitespace between the start tag and its contents).

Reply:
ah, I see. Thank you! I will move the whole element to next line 🙏